db/migrate/20161124170211_create_subnet_compute_ressources.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSubnetComputeRessources < ActiveRecord::Migration
+  def up
+    create_table :subnet_compute_resources do |t|
+      t.references :compute_resource
```

Comment:
is there a way to default to the taxonomy subnets instead of introducing a new db model?